src/Settings/SettingsAttribute/CustomRankingAttribute.php

```diff
@@ -0,0 +1,48 @@
+<?php
+
+/**
+ * This file is part of AlgoliaSearch Client PHP Helper.
+ *
+ * (c) Algolia Team <contact@algolia.com>
+ *
+ *  For the full copyright and license information, please view the LICENSE
+ *  file that was distributed with this source code.
+ */
+
+namespace Algolia\AlgoliaSearch\Helper\Settings\SettingsAttribute;
+
+use Algolia\AlgoliaSearch\Helper\Contracts\SettingContract;
+use Algolia\AlgoliaSearch\Helper\Helpers\Str;
+
+// @internal
```

Comment:
Can't we go:
```php
    /**
      * @internal
      */
```

Reply:
I’ve update php-cs-fixer config file sorry It update single comment with single line comment